lib/ansible/modules/cloud/amazon/route53.py

```diff
@@ -463,62 +475,58 @@ def main():
+            module.fail_json(msg="If you specify failover, region or weight you must also specify identifier")
 
     if command_in == 'create':
-        if ( weight_in is not None or region_in is not None or failover_in is not None ) and identifier_in is None:
-            module.fail_json(msg= "If you specify failover, region or weight you must also specify identifier")
-        elif ( weight_in is None and region_in is None and failover_in is None ) and identifier_in is not None:
-            module.fail_json(msg= "You have specified identifier which makes sense only if you specify one of: weight, region or failover.")
-
-
+        if (weight_in is not None or region_in is not None or failover_in is not None) and identifier_in is None:
+            module.fail_json(msg="If you specify failover, region or weight you must also specify identifier")
+        elif (weight_in is None and region_in is None and failover_in is None) and identifier_in is not None:
+            module.fail_json(msg="You have specified identifier which makes sense only if you specify one of: weight, region or failover.")
 
     if vpc_id_in and not private_zone_in:
-        module.fail_json(msg="parameter 'private_zone' must be true when specifying parameter"
-            " 'vpc_id'")
-
+        module.fail_json(msg="parameter 'private_zone' must be true when specifying parameter"" 'vpc_id'")
 
     # connect to the route53 endpoint
     try:
         conn = Route53Connection(**aws_connect_kwargs)
     except boto.exception.BotoServerError as e:
-        module.fail_json(msg = e.error_message)
+        module.fail_json(msg=e.error_message)
 
     # Find the named zone ID
     zone = get_zone_by_name(conn, module, zone_in, private_zone_in, hosted_zone_id_in, vpc_id_in)
 
     # Verify that the requested zone is already defined in Route53
     if zone is None:
         errmsg = "Zone %s does not exist in Route53" % zone_in
-        module.fail_json(msg = errmsg)
+        module.fail_json(msg=errmsg)
 
     record = {}
 
     found_record = False
     wanted_rset = Record(name=record_in, type=type_in, ttl=ttl_in,
-        identifier=identifier_in, weight=weight_in, region=region_in,
-        health_check=health_check_in, failover=failover_in)
-    for v in value_list:
+                         identifier=identifier_in, weight=weight_in,
+                         region=region_in, health_check=health_check_in,
+                         failover=failover_in)
+    for v in value_in:
         if alias_in:
             wanted_rset.set_alias(alias_hosted_zone_id_in, v, alias_evaluate_target_health_in)
         else:
             wanted_rset.add_value(v)
 
-    sets = invoke_with_throttling_retries(conn.get_all_rrsets, zone.id, name=record_in,
-                                          type=type_in, identifier=identifier_in)
+    sets = invoke_with_throttling_retries(conn.get_all_rrsets, zone.id, name=record_in, type=type_in, identifier=identifier_in)
```

Comment:
Please don't make lines longer! There was nothing really wrong with this line before